invenio_rdm_records/resources/resources.py

```diff
@@ -116,13 +116,16 @@ def review_delete(self):
     @request_headers
     @request_view_args
     @request_data
-    @response_handler()  # TODO: probably needs to change
+    @response_handler()
     def review_submit(self):
-        """Submit a draft for review."""
+        """Submit a draft for review or directly publish it."""
+        # TODO: check if we can parse it in the body parser or in a better place?
+        direct_publish = resource_requestctx.data.pop("direct_publish", False)
```

Comment:
why do we pop it?

Reply:
because is not part of the review payload. I thought is more clear to pop it and indicate its purpose i.e specify if we should try to direct publish or create a review request.